src/worker_pool/index.ts

```diff
@@ -111,6 +111,7 @@ export class WorkerInfo extends AsynchronouslyCreatedResource {
     postTask (taskInfo : TaskInfo) {
       assert(!this.taskInfos.has(taskInfo.taskId));
       assert(!this.terminating && !this.destroyed);
+      this.clearIdleTimeout();
```

Comment:
Let's move this within the `try/catch` before posting the message to the port.

Reply:
Moved.